tests/test_rpc.py

```diff
@@ -35,6 +35,8 @@ def test_getaddress(lianad):
     assert "address" in res
     # We'll get a new one at every call
     assert res["address"] != lianad.rpc.getnewaddress()["address"]
+    # new address has derivation_index higher than the previous one
+    assert res["derivation_index"] < lianad.rpc.getnewaddress()["derivation_index"]
```

Comment:
It should be equal to exactly `+2` right?

Reply:
the new one, yes I think